### ISSUE TITLE
Evaluate integration of Saucelabs in order to test cross-browser compatibility on commit

### ISSUE THREAD

**Issue by evilaliv3** [andyperlitch/jsbn#11]
It would be valuable to integrate with Saucelabs in order to track code coverage on commit.

As the library currently is missing any unit test / unit test framework, my suggestion is to start implemeting them using [grunt-saucelabs](https://github.com/axemclion/grunt-saucelabs) integrated with mocha.

This way it will be possiblle to automatically execute all the unit test on the selection of browsers targetted by the library (all) and know proactively where it is not working.

E.g.:
![screenshot-github com 2016-03-12 12-20-45](https://cloud.githubusercontent.com/assets/217034/13722634/f89d4fb2-e84c-11e5-9be7-353c14d139d5.png)

Good examples reference for doing this are at:
- https://github.com/dchest/scrypt-async-js
- https://github.com/openpgpjs/openpgpjs

It would be also nice to start tracking the code quality of the project using for example [CodeClimate](https://codeclimate.com/) tracking also the code coverage.

**Comment by andyperlitch** [comment]
Agreed. As of now there are no tests written, but I am going to try and find some time soon to at least set up the test suite with some initial tests. PRs are, of course, welcome.
